assisted-events-scrape/storage/cluster_events_storage.py

```diff
@@ -47,6 +52,21 @@ def store(self, component_versions, cluster, event_list):
             events = self.process_events(cluster_bash_data, event_list, event_names)
             self.store_events(events, only_new_events=False)
 
+    @classmethod
+    def _hash_user_name(cls, cluster: dict):
+        """mask user name with unique generated FNV-1a 128 bit id"""
+        if "user_name" not in cluster:
+            return
+
+        user_name = cluster.get("user_name")
+        del cluster["user_name"]
+
+        # empty username
+        if not user_name:
+            cluster["user_id"] = -1
```

Comment:
Well, we need to deiced what is the best value (integer) that describes missing user_name

Reply:
The reason why I suggest this is for usability: users are used to filter "not existing" by queries like `NOT _exists_` (this will work with unset field and null fields).
Making it integer will make of this attribute a special case where users will need to remember/investigate. It's not a big deal, but if possible it would be good to keep it consistent